meveo-admin-web/src/main/java/org/meveo/admin/jsf/converter/EntityReferenceConverter.java

```diff
@@ -66,6 +71,11 @@ public String getAsString(FacesContext context, UIComponent component, Object uu
 			return null;
 		}
 		
+		if (field.getEntityClazz().equals(User.class.getName())) {
```

Comment:
in fact you can be more generic, it is fine to use userService to display the username if the fieldClass is user, but you need also to addres all other cases (PRovider,...) since all those other cases are BAseEntity you could at least show the id

Reply:
I have updated two case . 
a) provider logic
b) CFT reference data type other than CET or user or provider --- I am showing id for this case.

Reply:
After remove list item from combo box, it will  show only 4 items. 

CET, User , provider, Repository, CustomEntityInstance.

![image](https://github.com/meveo-org/meveo/assets/9381810/76e34899-ffba-47b0-b386-e2487d3d5ecc)